### PR TITLE
feat: buttons

### PR DESCRIPTION
**Pull request by pankaj-prog**

### CODE REVIEW

**Review by j24m — APPROVED** [review]
good to merge .

File: src/components/button/button.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+}
+
+/* Rounded Corner Buttons  */
+.btn-rc {
+  border-radius: 6px;
+}
+
+/* Link Buttons */
+.btn-link-primary {
+  padding: 0px 1px 0px 1px;
```
Comment (rahulyadav139) [inline src/components/button/button.css:46]: you could have used shorthand property here.

**Review by rahulyadav139 — APPROVED** [review]

**Review by aiman-ansari — APPROVED** [review]
Everything is good to merge

**Review by tarunsankhla — APPROVED** [review]
Looks good

**Comment by ashishkatiyar2404** [comment]
your code looks good to me.
Easy to read to anyone and clean.
Having spaces only required that is good pankaj. Nice work😁

**Review by PuneetP16 — COMMENTED** [review]
Nice minimal design

**Comment by m26unkwn** [comment]
Love the Buttons.

I think there is some color problem with second link button 
It has white color same as background so it is not visible.

**Pull request merged**